modules/gltf/register_types.cpp

```diff
@@ -60,9 +60,9 @@ static void _editor_init() {
 	if (blend_enabled) {
 		Ref<DirAccess> da = DirAccess::create(DirAccess::ACCESS_FILESYSTEM);
 		if (blender3_path.is_empty()) {
-			WARN_PRINT(TTR("Blend file import is enabled in the project settings, but no Blender path is configured in the editor settings. Blend files will not be imported."));
+			WARN_PRINT("Blend file import is enabled in the project settings, but no Blender path is configured in the editor settings. Blend files will not be imported.");
 		} else if (!da->dir_exists(blender3_path)) {
-			WARN_PRINT(TTR("Blend file import is enabled, but the Blender path doesn't point to an accessible directory. Blend files will not be imported."));
+			WARN_PRINT("Blend file import is enabled, but the Blender path doesn't point to an accessible directory. Blend files will not be imported.");
 		} else {
```

Comment:
This part is editor code. The warning is shown in editor toast.

I added this TTR because every new user will be welcomed with this message. And possibly for every new project they create if they don't change the editor setting.

Reply:
It's only shown in the editor toast in `DEV_ENABLED` builds. Otherwise it would have to use `WARN_PRINT_ED`.

Reply:
It's still shown as a warning inside the Output panel. The output panel displays a yellow dot for it.

Users know something is wrong about the editor because there's a warning. But they won't know what it is. Then they post comments like "it's ridiculus that the Godot team says the version is stable even though the editor obviously goes wrong everytime it opens."

But yeah, people will likely translate the message before submitting a issue :P

Reply:
Yeah but that's a sign that this shouldn't be a `WARN_PRINT` like this. Users are already asking to disable Blender import by default because they don't want to have to deal with it: #82242. But that PR includes discussion for better solutions to this UX issue.

Reply:
I'm ok keeping this one translated as an exception until the UX is fixed though.

Reply:
Restored those two as `TTR`.